backend/src/install/mod.rs

```diff
@@ -124,6 +124,52 @@ pub async fn install(
         .json()
         .await
         .with_kind(crate::ErrorKind::Registry)?;
+    let mut db_handle = ctx.db.handle();
+    // TODO: Ask Aiden why this didn't work
+    // let manifests = stream::iter(crate::db::package::get_packages(&mut db_handle).await?)
+    //     .then(move |pkg| crate::db::package::get_manifest(&mut db_handle, &pkg))
+    //     .try_filter_map(|a| async { Ok(a) })
+    //     .try_collect::<Vec<Manifest>>();
```

Comment:
```suggestion
    // let manifests = stream::iter(crate::db::package::get_packages(&mut db_handle).await?)
    //     .then(|pkg| async move { crate::db::package::get_manifest(&mut db_handle, &pkg).await })
    //     .try_filter_map(|a| async { Ok(a) })
    //     .try_collect::<Vec<Manifest>>();
```
🙃

Reply:
dude....what?!?! I tried `move |x| async {...}` and that didn't work. Wthat the actual fuck is the difference?